pkg/webhooks/regularuser/regularuser.go

```diff
@@ -225,6 +231,18 @@ func isCustomDomainAuthorized(request admissionctl.Request) bool {
 		request.Kind.Group == customDomainGroup)
 }
 
+// isNetNamespaceAuthorized check if request is authorized for NetNamespace CR
+func isNetNamespaceAuthorized(request admissionctl.Request) bool {
+	fmt.Printf("%s\n", request.Name)
```

Comment:
We use `log`, but this seems leftover for development; let's kick it out.

Reply:
Yeah, this was leftover from debugging. Will remove.